### PR TITLE
[FEATURE] Add two snippets fibsTillNum and fibonacciTillNum

### PR DESCRIPTION
**Pull request by kriadmin**
## Description
Add two snippets fibsTillNum and fibonacciTillNum

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by kriadmin** [comment]
@iamsoorena  These are complex mathematical functions and I don't think we should disturb the users with them.

File: snippets/toKebabCase.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### toKebabCase
```
Comment (skatcat31) [inline snippets/toKebabCase.md]: This should be a separate PR

**Comment by kriadmin** [comment]
@iamsoorena Removed toKebabCase

**Comment by kriadmin** [comment]
@Chalarangelo in fibonacciUntilNum there is a typo.
It should be  `until n` instead of `until nth term` .
